src/pkg/projects/windowsdesktop/pkg/dir.props

```diff
@@ -21,6 +21,72 @@
     <GeneratePkg>false</GeneratePkg>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
```

Comment:
None right now, filed https://github.com/dotnet/core-setup/issues/6607. IMO this list should really be assembled from the transport packages or generated, but failing CI if there are changes will be good to prevent regressions in the immediate future.